codex-rs/tui/src/bottom_pane/command_popup.rs

```diff
@@ -31,8 +31,15 @@ pub(crate) struct CommandPopup {
 }
 
 impl CommandPopup {
-    pub(crate) fn new(mut prompts: Vec<CustomPrompt>) -> Self {
-        let builtins = built_in_slash_commands();
+    pub(crate) fn new(mut prompts: Vec<CustomPrompt>, skills_enabled: bool) -> Self {
+        let builtins = if skills_enabled {
+            built_in_slash_commands()
+        } else {
+            built_in_slash_commands()
+                .into_iter()
+                .filter(|(_, cmd)| *cmd != SlashCommand::Skills)
```

Comment:
can we simplify 

```suggestion
                .filter(|(_, cmd)|  skills_enabled || *cmd != SlashCommand::Skills)
```
and remove the if

Reply:
but also, should we use the existing is_visible method for this?